.github/workflows/pr-title-checker.yml

```diff
@@ -12,6 +12,6 @@ jobs:
   check:
     runs-on: ubuntu-latest
     steps:
-      - uses: Seagate/pr-title-checker@v1.3.4
```

Comment:
Seems we have not fetched tags from the remote repo.  https://github.com/thehanimo/pr-title-checker/tags 
It would be good to fetch them or create tags in the forked repo and use it. So that we can keep syncing the `main` branch with remote when required.

Reply:
@shailesh-vaidya, I tagged specific commit of `Seagate/pr-title-checker` as mentioned in original repo of GitHub Action and now it is working fine. no need merge this PR change.